### DISCUSSION TITLE
Image source not readable

### DISCUSSION THREAD

**Discussion by ramdoun87** [spatie/laravel-medialibrary#2339] [Q&A]
laravel 8 
php 8
spatie/laravel-medialibrary": "^9.0.0",

   'image_driver' => env('IMAGE_DRIVER', 'gd'),

When using watermark it gives me an error : Image source not readable

model : 
 ```
                    ->watermark(public_path('unnamed.png'))
                    ->watermarkOpacity(45)
                    ->watermarkPosition(Manipulations::POSITION_BOTTOM_LEFT)      // Watermark at the top
                    ->watermarkHeight(13, Manipulations::UNIT_PERCENT)    // 50 percent height
                    ->watermarkWidth(21, Manipulations::UNIT_PERCENT)
                    ->watermarkPadding(15)
```

error : 
Intervention\Image\Exception\NotReadableException

Image source not readable

Intervention\Image\AbstractDecoder::init
C:\xampp\htdocs\company\vendor\intervention\image\src\Intervention\Image\AbstractDecoder.php:346

**Comment by BaDcereal** [comment]
I have the same problem. I tried the GD and Imagick image drivers, with Imagick the image is manipulated and the watermark is applied but the opacity effect is not.

laravel v8.33.1
php 8.0.3
spatie/laravel-medialibrary 9.4.4
